src/main/java/io/github/mountainrange/mule/gameplay/WorldMap.java

```diff
@@ -5,19 +5,22 @@
 import io.github.mountainrange.mule.enums.MuleType;
 import javafx.geometry.Point2D;
 
+import java.awt.*;
+import java.io.Serializable;
 import java.util.*;
+import java.util.List;
 import java.util.stream.Collectors;
 
 /**
  * A class to represent the map and facilitates interactions between the actual data store and the rest of the program.
  */
-public class WorldMap<T extends Tile> implements Iterable<Tile> {
+public class WorldMap<T extends Tile> implements Iterable<Tile>, Serializable {
 
 	/** Backing instance of Grid. */
 	private Grid<T> baseGrid;
 
-	/** List of tiles owned by each player, sorted by the MULE installed on them. */
-	private Map<Player, EnumMap<MuleType, Set<Tile>>> productionTiles;
+	/* List of tiles owned by each player, sorted by the MULE installed on them */
+	private Map<Player, EnumMap<MuleType, Set<Point>>> productionTiles;
```

Comment:
Would it be possible to keep this just a Set<Tile>? We should probably make this transient, and just rebuild it after we finish loading `baseGrid` by looping through all the tiles once and putting the `Tile`s in the right `Set`s.